sdk/typescript/src/utils/suins.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const SUI_NS_NAME_REGEX = /^(?:(?:[a-z0-9][a-z0-9-]{0,62})\.?)*@[a-z0-9][a-z0-9-]{0,62}$/i;
+const SUI_NS_DOMAIN_REGEX = /^(?:(?:[a-z0-9][a-z0-9-]{0,62})\.?)+.sui$/i;
+const MAX_SUI_NS_NAME_LENGTH = 200;
```

Comment:
This will soon be 235 (on May's release), but since the current max is 68, we can make it 235 already.

Reply:
interesting, I copied this from some rust code, but 'll update it